event/monitoring.go

```diff
@@ -89,3 +92,177 @@ type PoolEvent struct {
 type PoolMonitor struct {
 	Event func(*PoolEvent)
 }
+
+// ServerAddress represents a server's address.
+// This type will be a string value of the server's canonical address.
+type ServerAddress string
+
+// TopologyID represents a unique topology.
+type TopologyID uuid.UUID
+
+// ServerKind represents the type of a server.
+type ServerKind uint32
+
+// These constants are the possible types of servers.
+const (
+	Standalone  ServerKind = 1
+	RSMember    ServerKind = 2
+	RSPrimary   ServerKind = 4 + RSMember
+	RSSecondary ServerKind = 8 + RSMember
+	RSArbiter   ServerKind = 16 + RSMember
+	RSGhost     ServerKind = 32 + RSMember
+	Mongos      ServerKind = 256
+)
+
+// String implements the fmt.Stringer interface.
+func (kind ServerKind) String() string {
+	switch kind {
+	case Standalone:
+		return "Standalone"
+	case RSMember:
+		return "RSOther"
+	case RSPrimary:
+		return "RSPrimary"
+	case RSSecondary:
+		return "RSSecondary"
+	case RSArbiter:
+		return "RSArbiter"
+	case RSGhost:
+		return "RSGhost"
+	case Mongos:
+		return "Mongos"
+	}
+
+	return "Unknown"
+}
+
+// TopologyKind represents a specific topology configuration.
+type TopologyKind uint32
+
+// These constants are the available topology configurations.
+const (
+	Single                TopologyKind = 1
```

Comment:
This was pulled from `description/topology_kind.go`, since those types are not included in the public API. Are you asking about the specific numbers they are assigned?